webapp/sources/rudder/rudder-web/src/main/elm/sources/Editor/ViewTechniqueTabs.elm

```diff
@@ -141,46 +155,63 @@ techniqueParameter model technique param =
-                                  in
-                                    TechniqueParameterModified param.id {param | constraints = newConstraint }), required True] []
-                      ]
-                    , div [ class " form-group" ] [
-                        label [ ] [ text "Value"]
-                      , div [class "input-group" ] [
-                          input [ readonly (not model.hasWriteRights), type_ "text",  class "form-control", value currentValue.value, placeholder "Value"
-                                , onInput (\s ->
+             let
+               displayValue =
+                 \index currentValue ->
+                    let
+                       (invalidEnumNameClass, invalidEnumName) =
+                         if (String.isEmpty (Maybe.withDefault "" currentValue.name)) then
+                           ("error",  ul [class "row"] [ li [ class "text-danger col-md-8" ] [ text "Enum name cannot be empty" ] ] )
+                         else
+                           ("", text "")
+                       (invalidEnumValueClass, invalidEnumValue) =
+                         if (String.isEmpty (currentValue.value)) then
+                           ("error",  ul [class "row"] [ li [ class "text-danger col-md-8" ] [ text "Enum value cannot be empty" ] ] )
+                         else
+                           ("", text "")
+                    in
+                    li[] [
+                      span [ class "border" ] []
+                    , div [ class "param" ] [
+                        div [ class "gm-labels left"] [
+                          div [ class "gm-label rudder-label gm-label-name "] [ text "Enum" ]
+                        ]
+                      , div [ class "form-group" ] [
+                          label [] [ text "Display name"]
+                        , input [ readonly (not model.hasWriteRights), type_ "text",  class ("col-xs-8 form-control " ++ invalidEnumNameClass), placeholder "Name"
+                                , value (Maybe.withDefault "" currentValue.name), placeholder "Name" , onInput (\s ->
                                     let
-                                      newConstraint = { constraint | select = Just (List.Extra.updateAt index (\val -> {val | value = s}) l  ) }
+                                      newConstraint = { constraint | select = Just (List.Extra.updateAt index (\val -> {val | name = if String.isEmpty s then Nothing else Just s}) l  ) }
                                     in
                                       TechniqueParameterModified param.id {param | constraints = newConstraint }), required True] []
+                        , invalidEnumName
+                        ]
+                      , div [ class " form-group" ] [
+                          label [ ] [ text "Value"]
+                        , div [class "input-group" ] [
```

Comment:
the "input-group" class is not used elsewhere in the form, it should be removed because it doesn't look the same as the "Display name"